src/linker/Linker.Dataflow/XmlFlowAnnotationSource.cs

```diff
@@ -120,44 +164,31 @@ public void ParseXml (string document)
 		private void ProcessAssemblies (LinkContext context, XPathNodeIterator iterator)
 		{
 			while (iterator.MoveNext ()) {
-				if (!ShouldProcessElement (iterator.Current)) {
+				if (!ShouldProcessElement (iterator.Current))
 					return;
-				}
 
-				AssemblyDefinition assembly = GetAssembly (context, GetAssemblyName (iterator.Current));
+				if (GetFullName (iterator.Current) == "*") {
+					foreach (AssemblyDefinition assemblyIterator in context.GetAssemblies ()) {
+						ProcessTypes (assemblyIterator, iterator);
```

Comment:
If I read this correctly:
* Let's say I specify to add an attribute `LinkerRemoveAttribute` to `NullableAttribute` on `assembly="*"`.
* The code will do ProcessTypes on all assemblies
* If one of those assemblies doesn't have `NullableAttribute` type, this will issue a warning

I think that for the `assembly="*"` case we should intentionally ignore if the specified types are missing. Otherwise there will be lot of warnings with no go way to get rid of them. Once we do find the type I think we should warn about everything else on the type, but the lack of the type itself should not cause a warning in the "All-assemblies" case.